test/powershell/engine/Jobs.Tests.ps1

```diff
@@ -0,0 +1,17 @@
+Describe 'Basic Job Tests' -Tags 'CI' {
+
+    $job = Start-Job {1}
+
+    It 'Test job creation' {
+        $job | should not be $null
+    }
+
+    It 'Test job State' {
+        Wait-Job $job -Timeout 5
+        $job.JobStateInfo.State -eq 'Completed' | should be $true
+    }
+
+    It 'Job output test' {
+        Receive-Job $job -wait | should be 1
+    }
+}
```

Comment:
I haven't seen a strong argument for requiring `EOL` at `EOF`, and it comes up often enough to be somewhat annoying and distracting.

If it's truly a problem for some tools, I think we should just run a tool over our source from time to time and fix it instead of asking each contributor every time we see it.